train.py

```diff
@@ -8,126 +7,25 @@
 
 import argparse
 
-import os
-import sys
-import random
-
 import torch
-from torch import cuda
 
 import onmt.opts as opts
+from train_multi import main as multi_main
+from train_single import main as single_main
```

Comment:
Any chance we could remove this as a separate file? Ideally train.py would have all the code for single_main and multi_main would be the only separate file. Want to have as few base level files as possible.